lib/rubocop/runner.rb

```diff
@@ -359,5 +365,12 @@ def get_processed_source(file)
         ProcessedSource.from_file(file, ruby_version)
       end
     end
+
+    def standby_team(config)
+      @team_by_config ||= {}
+      @team_by_config[config.object_id] ||= begin
```

Comment:
I think the `begin`..`end` is unnecessary here.

Reply:
Thanks! Fixed in https://github.com/rubocop-hq/rubocop/commit/e7fa9a916082e463ee803f979a5ceed55791c1c5